README.md

```diff
@@ -552,120 +569,153 @@ times a second. You can test the following properties:
 
-You'll probably want to be able to save your progress and reload it later, if
-your application fails or you need to abort the crawl for some reason. (Perhaps
-you just want to finish off for the night and pick it up tomorrow!) The
-`crawler.queue.freeze` and `crawler.queue.defrost` functions perform this task.
+It can be convenient to be able to save the crawl progress and later be able to
+reload it if your application fails or you need to abort the crawl for some
+reason. The `crawler.queue.freeze` and `crawler.queue.defrost` methods will let
+you do this.
 
-**A word of warning though** - they are not CPU friendly as they rely on
-JSON.parse and JSON.stringify. Use them only when you need to save the queue -
-don't call them every request or your application's performance will be incredibly
-poor - they block like *crazy*. That said, using them when your crawler commences
-and stops is perfectly reasonable.
+**A word of warning** - they are not CPU friendly as they rely on `JSON.parse`
+and `JSON.stringify`. Use them only when you need to save the queue - don't call
+them after every request or your application's performance will be incredibly
+poor - they block like *crazy*. That said, using them when your crawler
+commences and stops is perfectly reasonable.
 
-Note that the methods themselves are asynchronous, so if you are going to exit the
-process after you do the freezing, make sure you wait for callback - otherwise
-you'll get an empty file.
+Note that the methods themselves are asynchronous, so if you are going to exit
+the process after you do the freezing, make sure you wait for callback -
+otherwise you'll get an empty file.
 
 ```js
-// Freeze queue
-crawler.queue.freeze("mysavedqueue.json", function() {
+crawler.queue.freeze("mysavedqueue.json", function () {
     process.exit();
 });
 
-// Defrost queue
 crawler.queue.defrost("mysavedqueue.json");
 ```
 
 ## Cookies
 
-Simplecrawler now has an internal cookie jar, which collects and resends cookies
-automatically, and by default.
+simplecrawler has an internal cookie jar, which collects and resends cookies
+automatically and by default. If you want to turn this off, set the
+`crawler.acceptCookies` option to `false`. The cookie jar is accessible via
+`crawler.cookies`, and is an event emitter itself.
 
-If you want to turn this off, set the `crawler.acceptCookies` option to `false`.
-
-The cookie jar is accessible via `crawler.cookies`, and is an event emitter itself:
-
-### Cookie Events
+### Cookie events
 
 * `addcookie` (cookie)
 Fired when a new cookie is added to the jar.
 * `removecookie` (cookie array)
 Fired when one or more cookies are removed from the jar.
 
+## FAQ
+
+There are a couple of questions that pop up more often than others in the issue
+tracker. If you're having trouble with simplecrawler, please have a look at the
+list below before submitting an issue.
+
+- **Q: Why does simplecrawler discover so many crappy URL's?**
```

Comment:
Not at all, harmonizing the language throughout the docs should make it more readable!